File: 2015WorksApplications/internship/intern/Worksap/workspace/China_Intern/China_Intern_Java/src/jp/co/worksap/intern/entities/hotel/HotelMstDTO.java
package jp.co.worksap.intern.entities.hotel;

import jp.co.worksap.intern.entities.ICsvMasterDTO;

public class HotelMstDTO implements ICsvMasterDTO {
	/**
	 * 
	 */
	private static final long serialVersionUID = -3711231132225687846L;
	private Long hotelId;
	private String hotelName;
	private String address;
	private String tel;

	/**
	 * Default Constructor 
	 * 
	 * @param hotelId
	 * @param hotelName
	 * @param address
	 * @param tel
	 */
	public HotelMstDTO(Long hotelId, String hotelName, String address,
			String tel, int stars) {
		super();
		this.hotelId = hotelId;
		this.hotelName = hotelName;
		this.address = address;
		this.tel = tel;
	}

	public Long getHotelId() {
		return hotelId;
	}

	public String getHotelName() {
		return hotelName;
	}

	public String getAddress() {
		return address;
	}

	public String getTel() {
		return tel;
	}

}

File: 2015WorksApplications/internship/intern/Worksap/workspace/China_Intern/China_Intern_Cpp/src/entities/hotel/hotel_mst_dto.cpp
#include "hotel_mst_dto.h"
#include <iostream>
using namespace std;

HotelMstDTO::HotelMstDTO(long hotel_id, string hotel_name, string address,
		string tel, int stars) {
	this->hotel_id = hotel_id;
	this->hotel_name = hotel_name;
	this->address = address;
	this->tel = tel;
}

long &HotelMstDTO::get_hotel_id() {
	return hotel_id;
}

string &HotelMstDTO::get_hotel_name() {
	return hotel_name;
}

string &HotelMstDTO::get_address() {
	return address;
}

string &HotelMstDTO::get_tel() {
	return tel;
}

ostream &operator <<(ostream &out, HotelMstDTO &obj) {
	out << obj.hotel_id << " " << obj.hotel_name << " " << obj.address << " "
			<< obj.tel;
	return out;
}

istream &operator >>(istream &in, HotelMstDTO &obj) {
	in >> obj.hotel_id >> obj.hotel_name >> obj.address >> obj.tel;
	return in;
}

